server/views/partials/checkThisRecordMacro.njk

```diff
@@ -0,0 +1,53 @@
+
+{% macro checkThisRecordMacro(data) %}
```

Comment:
new macro to be used by /confirmArrival and /check-transfer

Reply:
The designs on these two pages are very different now - we won't want to share this.

Reply:
/confirmArrival reverted to previous state